ProjectLighthouse.Servers.GameServer/Controllers/LoginController.cs

```diff
@@ -138,7 +138,18 @@ public async Task<IActionResult> Login()
             user.LinkedRpcnId = npTicket.Platform == Platform.RPCS3 ? npTicket.UserId : 0;
             user.LinkedPsnId = npTicket.Platform != Platform.RPCS3 ? npTicket.UserId : 0;
             await this.database.SaveChangesAsync();
-                
+
+            if (DiscordConfiguration.Instance.DiscordIntegrationEnabled)
+            {
+                string announceMsg = DiscordConfiguration.Instance.RegistrationAnnouncement;
+                announceMsg = announceMsg.Replace("%user", username);
+                announceMsg = announceMsg.Replace("%instance", ServerConfiguration.Instance.Customization.ServerName);
+                announceMsg = announceMsg.Replace("%platform", npTicket.Platform.ToString());
+                await WebhookHelper.SendWebhook(title: "New user",
+                    description: announceMsg,
+                    dest: WebhookHelper.WebhookDestination.Registration);
+            }
```

Comment:
```suggestion
            {
                string userRegistrationMsg = DiscordConfiguration.Instance.RegistrationAnnouncement;
                userRegistrationMsg = userRegistrationMsg.Replace("%user", username);
                userRegistrationMsg = userRegistrationMsg.Replace("%instance", ServerConfiguration.Instance.Customization.ServerName);
                userRegistrationMsg = userRegistrationMsg.Replace("%platform", npTicket.Platform.ToString());
                await WebhookHelper.SendWebhook(title: "A new user has registered!",
                    description: userRegistrationMsg,
                    dest: WebhookHelper.WebhookDestination.Registration);
            }
```
A few nitpicks but otherwise LGTM, feel free to use (or not).